src/sentry/api/endpoints/organization_events_trends.py

```diff
@@ -22,7 +22,7 @@
 from sentry.utils.snuba import Dataset, raw_snql_query
 
 # converter is to convert the aggregate filter to snuba query
-Alias = namedtuple("Alias", "converter aggregate")
+Alias = namedtuple("Alias", "converter aggregate resolved_function")
```

Comment:
Nit: a `dataclass` gets you almost the same functionality you want (minus the destructuring) and gives you proper types on each of the values within.

Reply:
Agreed, but gonna leave it for now cause I don't wanna rewrite the json trends code